dvc/output.py

```diff
@@ -97,6 +99,7 @@ def loadd_from(stage, d_list):
                 live=live,
                 remote=remote,
                 **annot,
+                dir_meta=files,
```

Comment:
Let's just call it files for consistency? `dir_meta` is a little bit misleading, since it is really a tree object or an index, rather than just `Meta`.
```suggestion
                files=files,
```